browser-test/playwright.config.ts

```diff
@@ -27,6 +28,8 @@ export default defineConfig({
   use: {
     trace: 'on-first-retry',
     video: process.env.RECORD_VIDEO ? 'on-first-retry' : 'off',
+    // Fall back support config file until it is removed
+    baseURL: process.env.BASE_URL || BASE_URL // 'http://civiform:9000'
```

Comment:
Long term, settings will get migrated over here. The baseURL in particular should be set here as Playwright will automatically use that as the default.